system/ad_service_state_monitor/src/ad_service_state_monitor_node/state_machine.cpp

```diff
@@ -12,13 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include "ad_service_state_monitor/state_machine.hpp"
+
 #include <deque>
 #include <vector>
 
-#define FMT_HEADER_ONLY
-#include "ad_service_state_monitor/state_machine.hpp"
-
+#define FMT_HEADER_ONLY  // NOLINT
```

Comment:
Just out of curiosity, what's this `NOLINT` for?

Reply:
@kenji-miyake pre-commit fixes include order as below.
```
#define FMT_HEADER_ONLY
#include "ad_service_state_monitor/state_machine.hpp"

#include <fmt/format.h>
```
I felt this order is a little bit weird, so I added `NOLINT`

Reply:
I see, but I think it's okay to define all macros at the top of headers.